src/components/Map/Map.tsx

```diff
@@ -87,10 +92,33 @@ export const Map = ({
   layers = DefaultMapLayers,
   centerLocation = DefaultCenterLocation,
   zoom = DefaultZoom,
+  flyToZoomLevel = DefaultFlyToZoomLevel,
   markerLocation,
   markerIcon,
   onClick,
 }: MapProps) => {
+  const [map, setMap] = useState<LeafletMap | null>(null);
+
+  const validMarkerLocation = useMemo(() => {
+    if (!markerLocation?.latitude || !markerLocation?.longitude)
+      return undefined;
+    return markerLocation;
+  }, [markerLocation]);
```

Comment:
The reason for why it was done this way is because of how storybook handles setting the object.

When you click the "Set object"-button it immediatly sets the prop to an empty object without any properties.
This was causing leaflet to crash.

So by doing it this way it makes it possible to test the component inside storybook